app/controllers/application_controller.rb

```diff
@@ -1,10 +1,88 @@
 require './config/environment'
+require 'pry'
 
 class ApplicationController < Sinatra::Base
+    register Sinatra::ActiveRecordExtension
+    set :views, Proc.new { File.join(root, "../views/") }
 
-  configure do
-    set :public_folder, 'public'
-    set :views, 'app/views'
+    configure do
+      enable :sessions
+      set :session_secret, "secret"
+    end
+
+  get '/' do
+    erb :home
+  end
+
+  get '/index' do
+    erb :index
+  end
+
+  get '/tweets/new' do
+      @user = Helpers.current_user(session)
+      if Helpers.is_logged_in?(session) == false
+          redirect '/login'
+      else
+          erb :'tweets/new'
+      end
+  end
+
+  post '/tweets/new' do
+      if params[:content] == ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.
This applies anywhere you're creating or updating and checking that the input isn't empty.